docs/getting-started/cluster-setup.md

```diff
@@ -0,0 +1,232 @@
+# Setting Up a Cluster
+You can setup a cluster in [Kubernetes](http://kubernetes.io/docs/whatisk8s/) or Jenkins.
+
+## Kubernetes
+
+### Installing kubectl
+To deploy and manage applications on Kubernetes, you’ll use the Kubernetes command-line tool, __kubectl__. It lets you inspect your cluster resources, create, delete, and update components, and much more.
+
+#### Prerequisites
+- [brew](http://brew.sh/)
```

Comment:
Can we just point to k8s docs instead of trying to get this right and maintain it?